packages/app/backend-http-client/package.json

```diff
@@ -11,11 +11,7 @@
     "url": "git://github.com/lokalise/backend-http-client.git"
   },
   "license": "Apache-2.0",
-  "files": [
-    "dist/**",
-    "LICENSE.md",
-    "README.md"
-  ],
+  "files": ["dist/**", "LICENSE.md", "README.md"],
```

Comment:
I had to reformat this file. The CI Build was complaining about format of this file.